### PR TITLE
feat / Make more params configurable on the fly

### PR DESCRIPTION
**Pull request by vic-en**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue: #1716 
- Related Clubhouse Story:

**A description of the changes proposed in the pull request**:

### CODE REVIEW

File: hummingbot/strategy/pure_market_making/pure_market_making.pyx [hunk 1]
```diff
@@ -606,6 +606,9 @@ cdef class PureMarketMakingStrategy(StrategyBase):
             MarketBase market = self._market_info.market
             list buys = []
             list sells = []
+        # It is necessary to point buy and sell levels to order_levels in order to make the configurable on the fly
```
Comment (Nullably) [inline hummingbot/strategy/pure_market_making/pure_market_making.pyx:609]: You can do this in the order_levels property setter. Resetting the buy and sell levels here will be a problem for scripts.

**Review by Nullably — CHANGES_REQUESTED** [review]

**Review by Nullably — DISMISSED** [review]
LGTM

**Comment by RC-13** [comment]
Test the following parameters:
- `order_amount` OK
param-order-amount
param-order-amount1
- `order_levels` OK
param-order-level
- `order_level_spread` OK
param-order-spread
- `inventory_skew_enabled` OK
param-order-skew
- `inventory_target_base_pct` works with one line only
- Doing `config inventory_target_base_pct` then pressing enter doesn't give this question: `In [exchange], you have [base_asset_balance] and [quote_asset_balance]. By market value, your current inventory split is [base_%_ratio] and [quote_%_ratio]. Would you like to keep this ratio?` Instead, it automatically pull the current value of inventory_target_base_pct.
skew-base-pct
- `inventory_range_multiplier` OK
param-order-skew-multiplier
- `filled_order_delay` OK
confirmed working and changes applies after getting a filled order

**Review by dennisocana — CHANGES_REQUESTED** [review]
QA found a bug when trying to configure `inventory_target_base_pct` on the fly except in KuCoin, Coinbase Pro, Liquid and Bittrex.

**Comment by dennisocana** [comment]
@RC-13 please re-test this PR. As @vic-en mentioned, please note that only `inventory_target_base_pct` isn't configurable on the fly when using DEXs for now.

**Comment by RC-13** [comment]
Confirm `inventory_target_base_pct` on the fly is now working properly
image

**Review by dennisocana — APPROVED** [review]
QA tested and confirmed working.

**Review by dennisocana — APPROVED** [review]
LGTM

**Pull request merged**